### ISSUE TITLE
Add Repository.URL to SearchResult?

### ISSUE THREAD

**Issue by isker** [sourcegraph/zoekt#560]
Right now, [`SearchResult`](https://github.com/sourcegraph/zoekt/blob/52664b7dfdeb0ae3c198a393c9187f99fb3f8828/api.go#L469-L483) [provides URL templates](https://github.com/sourcegraph/zoekt/blob/52664b7dfdeb0ae3c198a393c9187f99fb3f8828/eval.go#L447-L457) for linking to individual files and line numbers within a repository, but not a URL for linking to the repository itself.  If you want to get the repository URL, you have to join a SearchResult to a list request, or try to heuristically extract it from the file URL template.

I'm sure adding fields to SearchResult is not to be done lightly, but what do you think about adding this one?

**Comment by isker** [comment]
Of course, adding such a field presents a naming dilemma.  `RepoURLs` is exactly what this would be - the existing thing should have been named something else.

**Comment by gogo2464** [comment]
did you mind about to use repo: ? not sure I understand the request.

**Comment by keegancsmith** [comment]
Yeah I don't think we will add support for this. I see a few requests for us to hydrate more interesting stuff into responses. Maybe if someone comes up with a creative way to do this in zoekt-webserver we can support this plus other use cases wanting to access Repository metadata/etc.
